app/import.go

```diff
@@ -52,7 +52,7 @@ func processAttachmentPaths(files []*imports.AttachmentImportData, basePath stri
 			*f.Path = path
 			if len(filesMap) > 0 {
 				if files[i].Data, ok = filesMap[path]; !ok {
-					mlog.Warn(fmt.Sprintf("attachment %s not found and will not be imported", path))
+					mlog.Warn("Warn message", mlog.String(fmt.Sprintf("attachment %s not found and will not be imported", path), ""))
```

Comment:
there should a method name Warnf in the mlog library which can format the log as well see if you can use it

Reply:
but Warnf is nowhere used in mattermost server

Reply:
Warnf is not declared by package mlog

Reply:
so what ? if its useful here then use it